torch/_dynamo/variables/torch.py

```diff
@@ -323,6 +322,12 @@ def call_function(
             return DisabledSavedTensorsHooksVariable.create(
                 tx, args[0].as_python_constant()
             )
+        elif self.value is torch._functorch.vmap.vmap_nesting:
+            assert len(args) == 2
+            return VmapCtxManagerVariable.create(
+                tx,
+                [guard_if_dyn(x) for x in args],
```

Comment:
What's this?

Reply:
`guard_if_dyn` gets the underlying python constant variable, which in this case are the `batch_size` and `randomness` values.